### PR TITLE
Adds vara-cs package utility tool

### PR DESCRIPTION
**Pull request by vulder**
This commit implements the new result package creator vor vara-cs. The package tool compresses a paper config with its results into a small zip archive, which can be shared and unzipped to allow analysis on another system.

resolves se-passau/VaRA#435

### CODE REVIEW

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/paper/paper_config_manager.py  2
- varats/driver.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VaRA/VaRA-Tool-Suite/pullRequest?prid=3758942&bid=13194703)

File: varats/driver.py [hunk 1]
```diff
@@ -535,6 +545,19 @@ def add_common_args(sub_parser: argparse.ArgumentParser) -> None:
...
+            if Path(os.getcwd()) != vara_root:
+                print("Packaging needs to be called from VaRA root dir,"
+                      " changing dir to {vara_dir}".format(vara_dir=vara_root))
+                os.chdir(vara_root)
+
+            import re
+            PCM.package_paper_config(
+                Path(args["output"]), re.compile(args['filter_regex']))
+        else:
+            parser.error("--output needs to be a zip file path, e.g., foo.zip")
```
Comment (julianbreiteneicher) [inline varats/driver.py]: I would suggest the following:
  - Output this error message if the user provides a filename with a different extension than
    '.zip', e.g. '.bar'.
  - If the user doesn't provide an extension at all (e.g. `args['output'] == "foo"`), then you can 
    simply add the .zip extension without an error message

**Review by julianbreiteneicher — APPROVED** [review]
Only one small suggestion. Otherwise looks good!

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/paper/paper_config_manager.py  2
- varats/driver.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VaRA/VaRA-Tool-Suite/pullRequest?prid=3758942&bid=13194703)

**Review by labrenz — APPROVED** [review]

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/paper/paper_config_manager.py  2
- varats/driver.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VaRA/VaRA-Tool-Suite/pullRequest?prid=3758942&bid=13194703)

**Pull request merged**